README.md

```diff
@@ -27,6 +27,12 @@ Run the application by executing:
 ./gradlew bootRun
 ```
 
+### Operational note
```

Comment:
even operationally to support the user, at the request of the user?

Reply:
I think at that point it gets a little more complex in terms of business process as it would need to be qualified staff and some form of official record that the citizen had given express permission.

It's a valid requirement I think we should bring up with Data and Security to see how we can facilitate it.

Reply:
would be great if this was explored as do we have these problems...

Reply:
copied you into an email about it 👍